ts/screens/showroom/Showroom.tsx

```diff
@@ -56,15 +57,17 @@ export const Showroom = () => {
             <NBView spacer={true} large={true} />
             <TextFieldsShowroom />
             <NBView spacer={true} large={true} />
+            <ListItemsShowroom />
+            <NBView spacer={true} large={true} />
             <ToastNotificationsShowroom />
             <NBView spacer={true} large={true} />
             <PictogramsShowroom />
             <NBView spacer={true} large={true} />
             <IconsShowroom />
             <NBView spacer={true} large={true} />
             <LogosShowroom />
-            <NBView spacer={true} large={true} />
-            <IllustrationsShowroom />
+            {/* <NBView spacer={true} large={true} />
+            <IllustrationsShowroom /> */}
```

Comment:
Does this need to be commented or can we remove it directly?

Reply:
It's better not to remove it because we can use the section as reference when we start to replace all the legacy assets. Probably we're gonna move them in legacy pages once we break the UI Showroom in different subpages.